src/server_manager/install_scripts/install_server.sh

```diff
@@ -81,18 +94,59 @@ function log_for_sentry() {
 # Check to see if docker is installed.
 function verify_docker_installed() {
   if ! command_exists docker; then
-    log_error "Docker CE must be installed, please run \"curl -sS https://get.docker.com/ | sh\" or visit https://docs.docker.com/install/"
-    exit 1
+    log_error "FAILED"
+    echo -n "> Would you like to install Docker? [Y/n] "
+    if ! run_step_with_user_confirmation "Installing Docker" install_docker; then
+      log_error "Docker installation failed, please visit https://docs.docker.com/install for instructions."
+      exit 1
+    fi
+    echo -n "> Verifying Docker installation................ "
+    command_exists docker
```

Comment:
Should you call `run_step` here?

Reply:
The issue with calling `run_step` within a previous call is that we get a trailing `echo "OK"`. Avoiding it here ensures the final "OK" is printed in the right location.